README.md

```diff
@@ -62,23 +95,34 @@ Ensure all ROS 2 prerequisites are fulfilled,
 you can subsutitute your distro name for `<your ros distro>`
 
 Example:
-for `galactic`
+for `humble`
 
 ```bash
 cd ~/rmf_ws
-rosdep install --from-paths src --ignore-src --rosdistro galactic -y
+rosdep install --from-paths src --ignore-src --rosdistro humble -y
 ```
 
 ### Compiling Instructions
 
-> NOTE: Due to newer changes in the source build, there might be conflicts and compilation errors with older header files installed by the binaries. Please remove the binary installations before building from source, using `sudo apt remove ros-galactic-rmf*`.
+> NOTE: Due to newer changes in the source build, there might be conflicts and compilation errors with older header files installed by the binaries. Please remove the binary installations before building from source, using `sudo apt remove ros-humble-rmf*`.
+
+Compiling on `Ubuntu 22.04`:
+
+#### Install clang
+
+```bash
+sudo apt update
+sudo apt install clang lldb lld
+```
+
+**NOTE: RMF does not support building on gcc.**
 
-Compiling on `Ubuntu 20.04`:
+#### Compile using clang
 
 ```bash
 cd ~/rmf_ws
-source /opt/ros/<your ros distro>/setup.bash
-colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release
+source /opt/ros/humble/setup.bash
+CXX=clang++ LDFLAGS='-fuse-ld=lld' colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release
```

Comment:
We should promote using`colcon mixin`. It's a great tool that users will benefit from.

```suggestion
# Update colcon mixin which is one time step
colcon mixin add default https://raw.githubusercontent.com/colcon/colcon-mixin-repository/master/index.yaml
colcon mixin update default

export CXX=clang++
export CX=clang
colcon build --mixin "release lld"
```